src/lib/create-config.ts

```diff
@@ -267,6 +267,7 @@ const defaultAuthentication: IAuthOption = {
     initialAdminUser: {
         username: process.env.UNLEASH_DEFAULT_ADMIN_USERNAME ?? 'admin',
         password: process.env.UNLEASH_DEFAULT_ADMIN_PASSWORD ?? 'unleash4all',
+        email: process.env.UNLEASH_DEFAULT_ADMIN_EMAIL ?? 'admin@unleash.com',
```

Comment:
1. unleash.com is not associated with the Unleash project. 
2. It should be undefined if not set. We should not force it on everyone.